packit.spec

```diff
@@ -46,8 +46,10 @@ Requires:       fedpkg
 Requires:       rsync
 # bumpspec
 Requires:       rpmdevtools
-# See setup.cfg for details
+%if 0%{?rhel}
```

Comment:
just to be clear, the reason for this condition is that the python dep generator doesn't work for el8, right?

Reply:
No, it's there because the bug has been fixed in fedora only, i.e. koji in epel8 still doesn't provide egg-info, hence we need to have explicit Requires there.

Reply:
uhhhhhhhhh

Reply:
I've updated the commit message and added comments in spec. Thanks for the review.